content/Wrath.BPCoreModTemplate/{SourceName}/{SourceName}.csproj

```diff
@@ -24,7 +24,9 @@
 	</ItemGroup>
 	<ItemGroup>
 		<None Include="Info.json" CopyToOutputDirectory="PreserveNewest" />
-		<EmbeddedResource Include="LocalizedStrings.json" />
+		<None Update="LocalizedStrings.json">
+			<CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+		</None>

```

Comment:
Line above:

    <None Include="Info.json" CopyToOutputDirectory="PreserveNewest" />

Line below:

    <None Update="LocalizedStrings.json">
        <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
    </None>

Reply:
Blame VS for autoformatting

Reply:
I do blame it a lot.
It tries to use 2 whitespace by default in a csproj. Imagine the horror.